server/table.go

```diff
@@ -121,6 +124,20 @@ func (t *Table) getColumnsByName(names []string) ([]*Column, error) {
 	return columns, nil
 }
 
+func (t *Table) addForeignKeyConstraint(stmt *ast.CreateTable, parentStmt *ast.CreateTable) {
+	if stmt.Cluster != nil && parentStmt != nil {
+		var columns string
+		for idx, key := range parentStmt.PrimaryKeys {
+			if idx == 0 {
+				columns += fmt.Sprintf("%s", key.Name.Name)
+			} else {
+				columns += fmt.Sprintf(", %s", key.Name.Name)
+			}
+		}
+		t.foreignKeyConstraint = fmt.Sprintf("FOREIGN KEY(%s) REFERENCES %s(%s) %s", columns, parentStmt.Name.Name, columns, stmt.Cluster.OnDelete)
```

Comment:
`stmt.Cluster.OnDelete` is possibly nil. so it's better to check non-nil or switch for each case

Reply:
It can't be nil I think, it's of type `OnDeleteAction` which is just a string extension so it's nil value is empty string right?

![image](https://user-images.githubusercontent.com/18051009/72128739-32a61b80-33b7-11ea-94cf-e36b648bc99f.png)

Reply:
at least if I try to check `!= nil` my IDE says that comparison is not valid for string type

Reply:
ah it's string so can be nil but possibly empty string? because it is described as optional in the comment.

Reply:
yes it can't be nil, only empty string so in either case using it directly shouldn't result in any weird behavior. Confirmed in my latest commit as well.